watson_speech_to_text_block.py

```diff
@@ -44,6 +45,6 @@ def process_signals(self, signals):
                 self.logger.exception("Failed to open speech file: ")
```

Comment:
Originally I wasn't planning on putting that in and just leaving it as a general error but I think it would be good to give an explicit message of the location it can't find